src/compas_eve/core.py

```diff
@@ -58,6 +61,150 @@ def unadvertise(self, topic):
         pass
 
 
+class MessageCodec:
```

Comment:
In order to maintain IronPython compat, the class def should explicitly inherit from `object`: 
```suggestion
class MessageCodec(object):
```

Once we move fully to cpython, we can simplify this again.